game-core/src/main/java/games/strategy/triplea/TripleA.java

```diff
@@ -32,41 +31,29 @@
 public class TripleA implements IGameLoader {
   private static final long serialVersionUID = -8374315848374732436L;
 
-  protected transient ITripleADisplay display;
   protected transient IGame game;
-  private transient ISound soundChannel;
 
   @Override
-  public Set<IGamePlayer> newPlayers(final Map<String, PlayerType> playerNames) {
+  public Set<IRemotePlayer> newPlayers(final Map<String, PlayerType> playerNames) {
     return playerNames.entrySet().stream().map(TripleA::toGamePlayer).collect(Collectors.toSet());
   }
 
-  private static IGamePlayer toGamePlayer(final Map.Entry<String, PlayerType> namePlayerType) {
+  private static IRemotePlayer toGamePlayer(final Map.Entry<String, PlayerType> namePlayerType) {
     return namePlayerType.getValue().newPlayerWithName(namePlayerType.getKey());
   }
 
   @Override
   public void shutDown() {
-    if (game != null && soundChannel != null) {
-      game.removeSoundChannel(soundChannel);
-      // set sound channel to null to handle the case of shutdown being called multiple times.
-      // If/when shutdown is called exactly once, then the null assignment should be unnecessary.
-      soundChannel = null;
-    }
-
-    if (display != null) {
-      if (game != null) {
-        game.removeDisplay(display);
-      }
-      display.shutDown();
```

Comment:
Is this shutdown a no-op? Why can it be removed? Is it located somewhere else now?

Reply:
Yeah, I made shutdown part of AbstractGame, so the display is shut down whenever the display is set to null.

Reply:
Side note: `TripleADisplay#shutDown` actually calls `TripleAFrame#shutDown`, so without this call the game window wouldn't close after the game stops.